plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -20,6 +20,12 @@ const pipelineStepCompletionCounter = new Counter({
     help: 'Number of events that have completed the step',
     labelNames: ['step_name'],
 })
+const pipelineStepDurationSummary = new Summary({
+    name: 'events_pipeline_step_duration_seconds_total',
+    help: 'Duration spent in each step',
+    percentiles: [0.1, 0.5, 0.9, 0.95],
+    labelNames: ['step_name'],
```

Comment:
Should we have `teamId` as a label too? Maybe too much cardinality...

Reply:
too much cardinality to add to a dashboard. as we decided to go forward with otel tracing, we'll have that info in the traces instead